ceno_zkvm/benches/fibonacci.rs

```diff
@@ -70,7 +70,7 @@ fn fibonacci_prove(c: &mut Criterion) {
                 "max_steps = {}, proof size = {}, hashes count = {}",
                 max_steps,
                 serialize_size,
-                stat_recorder.borrow().field_appended_num
+                stat_recorder.into_inner().field_appended_num
```

Comment:
This `.into_inner()` enlists the borrow checker into making sure that no one else holds any access to our `stat_recorder` refcell anymore.  Specifically, this `.into_inner()` only works when the borrowing in `BasicTranscriptWitStat::new(&stat_recorder, b"riscv");` is done.